types/underscore/underscore-tests.ts

```diff
@@ -3514,116 +3251,371 @@ _.identity({ name: 'moe' }); // $ExpectType { name: string; }
 
 // constant
 
-// creating a function that will always return a specific value
-_.constant({ name: 'moe' }); // $ExpectType () => { name: string; }
+// creating a function that will always return a specific value
+_.constant({ name: 'moe' }); // $ExpectType () => { name: string; }
```

Comment:
Curious diff artifact.

Reply:
Yeah, not sure how git arrived at that, though it seems to have gone away when making these last few commits.